### ISSUE TITLE
Add "Recently Listed" ("Newest Added") list to NFTs landing page

### ISSUE THREAD

**Issue by fvictorio** [0xProject/0x-launch-kit-frontend#324]
Depends on #318 (and it might make sense to do it after #320).

**Comment by unjapones** [comment]
Updated the issue title and description.

The "Most traded" list/feature will be moved to another issue, since that involves checking the log/on-chain information.
